pybamm/solvers/solution.py

```diff
@@ -14,8 +14,9 @@
 
 class NumpyEncoder(json.JSONEncoder):
     """
-    Numpy serialiser helper class that converts numpy arrays to a list
-    https://stackoverflow.com/questions/26646362/numpy-array-is-not-json-serializable
```

Comment:
Ditto!

Reply:
I added plain text that explains, so we don't need the link anymore. If the explanation is not good enough then we can just update the text